docs/changelog/1_0_b3.rst

```diff
@@ -0,0 +1,203 @@
+* Enforce :eql:type:`duration` constraints in SQL as opposed to only
+  during casting (:eql:gh:`#2539`).
+* Fix how nested volatile computed expressions get executed
+  (:eql:gh:`#2545`).
+* Fix how ``USING`` expressions propagate to subtypes (:eql:gh:`#2543`).
+
+
+GraphQL
+=======
+
+* Remove the helper ``stdgraphql`` module as it is no loger needed
+  (:eql:gh:`#2692`).
+* Fix an issue with inline fragments (:eql:gh:`#1800`).
+
+
+Command-Line Tools
+==================
+
+We've changed some of the CLI `commands and groupings <rfc1006_>`_.
+
+.. list-table::
+    :widths: auto
+    :header-rows: 1
+
+    * - SUBCOMMAND
+      - DESCRIPTION
+    * - ``authenticate``
+      - Authenticate to a remote instance
+    * - ``dump``
+      - Create a database backup
+    * - ``restore``
+      - Restore a database backup from file
+    * - ``configure``
+      - Modify database configuration
+    * - ``migration apply``
+      - Bring current database to the latest or a specified revision
+    * - ``migration create``
+      - Create a migration script
+    * - ``migration status``
+      - Show current migration state
+    * - ``migration log``
+      - Show all migration versions
+    * - ``migrate``
+      - An alias for edgedb migration apply
+    * - ``database create``
+      - Create a new DB
+    * - ``describe object``
+      - Describe a database object
+    * - ``describe schema``
+      - Describe schema of the current database
+    * - ``list``
+      - List matching database objects by name and type
+    * - ``server info``
+      - Show locally installed EdgeDB servers
+    * - ``server install``
+      - Install an EdgeDB server locally
+    * - ``server uninstall``
+      - Uninstall an EdgeDB server locally
+    * - ``server upgrade``
+      - Upgrade installations and instances
+    * - ``server list-versions``
+      - List available and installed versions of the server
+    * - ``instance create``
+      - Initialize a new server instance
+    * - ``instance status``
+      - Show statuses of all or of a matching instance
+    * - ``instance start``
+      - Start an instance
+    * - ``instance stop``
+      - Stop an instance
+    * - ``instance restart``
+      - Restart an instance
+    * - ``instance destroy``
+      - Destroy an instance and remove the data
+    * - ``instance logs``
+      - Show logs of an instance
+    * - ``instance revert``
+      - Revert a major instance upgrade
+    * - ``project init``
+      - Initialize a new or existing project
+    * - ``project unlink``
+      - Clean-up the project configuration
+    * - ``project info``
+      - Get various metadata about the project
+    * - ``project upgrade``
+      - Upgrade EdgeDB instance used for the current project
+    * - ``self upgrade``
+      - Upgrade this edgedb binary
+    * - ``self install``
+      - Install server
+
+* Implemented `RFC 1008: TLS and ALPN <rfc1008_>`_.
+* Add an option to generate self-signed certificates.
```

Comment:
self-signed certs are part of TLS and ALPN work, no need to break out a separate item.  Also, I'd add a blurb from the motivation section of the RFC here to explain why we are adding TLS: it is to properly support protocol selection, but also proper transport security by default.

Reply:
I've reworked the TLS section.